spacewalk/config/var/lib/rhn/rhn-satellite-prep/etc/rhn/rhn.conf

```diff
@@ -2,7 +2,6 @@ traceback_mail = @@traceback_mail@@
 mount_point = @@mount_point@@
 kickstart_mount_point = @@kickstart_mount_point@@
 repomd_cache_mount_point = /var/cache 
-server.satellite.rhn_parent = satellite.rhn.redhat.com
```

Comment:
Are we dropping `server.satellite.rhn_parent` from `rhn.conf` but not from other conf files like proxy?

Reply:
This is the setup program for the server. So we drop this from the server as the server to server connection is the old redhat cdn connection which we never used.
On a proxy the rhn_parent is used in case of ISS and refer to the Uyuni/SUSE Manager Server.
So we need to keep it.